packages/commonwealth/server/workers/evmChainEvents/nodeProcessing.ts

```diff
@@ -72,6 +97,60 @@ export async function processChainNode(
         await lastProcessedBlock.save({ transaction });
       }
 
+      const contestEvents = events
```

Comment:
This should use `allEvents` rather than `events` since `allEvents` will include events that may have been migrated.